eventsource/dedicated-worker/eventsource-constructor-no-new.js

```diff
@@ -0,0 +1,7 @@
+test(function() {
```

Comment:
This file won't be recognized a test, you'll need to name it .any.js or .window.js

Reply:
Thanks @foolip